### PR TITLE
Eh 897 arvioijat

### PR DESCRIPTION
**Pull request by vesaruu**
Added aiemminHankitunOsaamisenArvioijat to tutkinnon osa details. Refactored some tests.

### CODE REVIEW

File: shared/components/StudyInfo/Details.tsx [hunk 1]
```diff
@@ -73,9 +74,14 @@ const CollapsedDetailsTitle = styled("strong")`
   margin: 10px 0 8px 0;
 `
 
-const AiemmanOsaamisenTitleExpanded = styled("strong")`
+const ExpandedDetailsTitle = styled("strong")`
   display: block;
-  margin: 10px 0 8px 20px;
+  margin: 10px 0 8px;
+  margin-left: ${props => props.theme.spacing.l};
```
Comment (J-Kallunki) [inline shared/components/StudyInfo/Details.tsx:80]: Example for that https://github.com/Opetushallitus/ehoks-ui/blob/master/shared/components/StudyInfo/OsaamisenOsoittaminen.tsx#L40

File: shared/components/StudyInfo/Details.tsx [hunk 2]
```diff
@@ -330,66 +330,114 @@ const TodentamisenProsessiCollapsed = ({
   todentamisenProsessi?: TodentamisenProsessi
 }) => (
   <>
-    <TodentamisenProsessiSuoraan
-      todentamisenProsessiKoodi={todentamisenProsessiKoodi}
-    />
+    {todentamisenProsessiKoodi === TodentamisenProsessiKoodi.SUORAAN && (
```
Comment (J-Kallunki) [inline shared/components/StudyInfo/Details.tsx:333]: `=> todentamisenProsessiKoodi === TodentamisenProsessiKoodi.SUORAAN && (`

File: shared/components/StudyInfo/Details.tsx (hunk 1 above)
Comment (vesaruu) [reply shared/components/StudyInfo/Details.tsx]: Changed

File: shared/components/StudyInfo/Details.tsx (hunk 2 above)
Comment (vesaruu) [reply shared/components/StudyInfo/Details.tsx:327]: I think there needs to be react.fragment as there is also todentamisenProsessiKoodi ===  TodentamisenProsessiKoodi.ARVIOIJIEN_KAUTTA  ?

**Pull request merged**